raco/backends/myria/myria.py

```diff
@@ -1481,14 +1470,15 @@ def fire(self, op):
 
 class PushIntoSQL(rules.Rule):
 
-    def __init__(self, dialect=None):
+    def __init__(self, dialect=None, push_grouping=False):
```

Comment:
Is there any long-term reason why aggregates should be treated any differently from projections, selections, or joins WRT pushing into SQL, or is this just short-term risk mitigation?